src/views/Conference.vue

```diff
@@ -17,15 +18,23 @@ export default {
     Page,
   },
   props: {
-    room: { type: String, required: true },
+    room: { type: String, default: '' },
   },
-  mounted() {
+  async mounted() {
+    const connection = await BrowserWindowMessageConnection({
+      origin: 'https://localhost:8080',
+    });
+
+    connection.connect(({ room }) => {
+      this.$router.push({ name: 'conference', params: { room } });
+    });
```

Comment:
I suggest to don't use this api and do it by ourselves. Actually, it is just `window.addEventListener('message', ...)`

Reply:
@davidyuk true, but I think that to use the same `BrowserWindowMessageConnection` makes code (how to say that)  more consistency.

Reply:
Also I didn't import the new dependency for that and use the same at the jitsi side for communion with wallet and superhero-ui . So I think to leave that as it is for the better consistency.

Reply:
@davidyuk I looked at https://github.com/aeternity/aepp-sdk-js/blob/47179f7a30c39b59110ebec82919ce219367f4b1/es/utils/aepp-wallet-communication/connection/browser-window-message.js and think that you are right. BrowserWindowMessageConnection instance can take more memory.

Reply:
I will leave `BrowserWindowMessageConnection